scripts/test_api.py

```
@@ -0,0 +1,83 @@
+import requests
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)